server/form-pages/apply/basic-information/releaseType.ts

```diff
@@ -14,7 +14,7 @@ const allReleaseTypes = {
 } as const
 
 type AllReleaseTypes = typeof allReleaseTypes
-type ReducedReleaseTypes = Pick<AllReleaseTypes, 'rotl' | 'license' | 'rerelease'>
```

Comment:
This commit has caused the tests to fail. I think the lines that mention`rerelease` need to be removed from this file: https://github.com/ministryofjustice/hmpps-approved-premises-ui/blob/ce51fc7a13c78c8b4a3d0c4bb57e0dbdef0e90e9/server/form-pages/apply/basic-information/releaseType.test.ts